### DISCUSSION TITLE
Incorporating dropdowns

### DISCUSSION THREAD

**Discussion by vaccine** [janosh/svelte-toc#60] [Ideas]
I think this would be a good optional feature. This would allow for a cleaner layout and less clutter.  Examples below:

Retracted:
![image](https://github.com/user-attachments/assets/b3eb2dfd-30c0-4183-87fd-3cc04535466b)

Extended:
![image](https://github.com/user-attachments/assets/18316787-4ccf-4941-acf9-67368b0ef691)

**Comment by janosh** [comment]
yes that would be a nice feature! happy to help anyone who wants to submit a PR
